pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors"}]
+description = "Distributed image processing"
+readme = "README.md"
+license = { file="LICENSE.txt" }
+dynamic = ["version"]
```

Comment:
When you `pip install .` it says:
```
Successfully installed dask-image-0.0.0
```
but when you open ipython and check the version string there, it is as expected
```
import dask_image
dask_image.__version__
# '2022.09.0+31.g39c9e19.dirty'
```

Possibly this has something to do with the dynamic versioneer stuff. Is this ok?
It's possible this won't be a problem when you install a released version from PyPI or conda-forge, but I haven't checked.

Reply:
More of a problem - dask-image version 0.0.0 is what shows up in the `conda list`. Since that is what gets shared during debugging, that's not great.

Reply:
[This post](https://github.com/pypa/setuptools_scm/issues/636#issuecomment-1192159170) suggests that having `setuptools>=62.6` installed and `setup_requires=["setuptools_scm"]` could potentially fix this problem?

Reply:
Thank you!
Now we have:
* `2022.9.1.dev33+gb43d1f9.d20230802` in conda list, and
* `2022.09.0+33.gb43d1f9.dirty` from ` import dask_image; dask_image.__version__`